src/lit/video.test.ts

```diff
@@ -0,0 +1,64 @@
+import {assert, expect, fixture, html, aTimeout} from '@open-wc/testing';
+import {DeguVideo} from './video';
+import * as dom from '../dom/dom';
+import * as func from '../func/func';
+
+// Define it.
+!window.customElements.get('degu-video') &&
+  window.customElements.define('degu-video', DeguVideo);
+
+const addStylesToPage = func.runOnlyOnce(() => {
+  dom.addStylesToPage(`
+    degu-video {
+      display: inline-block
```

Comment:
Small nit but I think this is missing semicolons at the end, and fix the inconsistent indentation

Reply:
Done